code/modules/mob/mob.dm

```diff
@@ -668,6 +668,9 @@ var/next_mob_id = 0
 		var/mob/living/L = src
 		if(L.has_status_effect(/datum/status_effect/freon))
 			canmove = 0
+	if(!lying && lying_prev)
+		if(client)
+			client.move_delay = world.time + movement_delay()
```

Comment:
Won't this get a bit weird since it will be affected by the mob's move speed(a person in no grav with a back jetpack will have no rest delay, while a guy in no grav with a hardsuit built-in jetpack will have a deay).